projects/vdk-plugins/vdk-dag/tests/test_dag.py

```diff
@@ -350,7 +406,25 @@ def test_dag_concurrent_running_jobs_limit(self):
             # assert that all the jobs finished successfully
             assert len(running_jobs) == 0
             self.httpserver.stop()
-    """
+
+    def test_dag_execution_type_propagation(self):
```

Comment:
This is testing the happy path. Are there tests covering the error paths?

Reply:
We decided to leave this one for later.